colibre/auto_plotter/black_holes.yml

```diff
@@ -293,10 +298,11 @@ stellar_veldisp_black_hole_mass_30:
       units: km/s
   metadata:
     title: LOS Stellar Velocity Dispersion-Black Hole Mass relation (30 kpc)
-    caption: The 3D stellar velocity dispersion is converted into a LOS velocity dispersion using a $1/\sqrt{3}$ correction factor. Note that the stellar velocity dispersion in observations is measured in a fixed 1 kpc aperture.
+    caption: The 3D stellar velocity dispersion is converted into a LOS velocity dispersion using a $1/\sqrt{3}$ correction factor. The stellar velocity dispersion from Sahu et al. (2016) was measured using a fixed 1 kpc aperture, while that from Saglia et al. (2016) was measured using the half-light radius aperture.
     section: Black Holes
     show_on_webpage: false
   observational_data:
+    - filename: StellarVelocityDispersionBlackHoleMass/Saglia2016.hdf5
```

Comment:
I don't think the version of the submodule `velociraptor-comparison-data` in this repository has been updated recently. This means that the current version of `velociraptor-comparison-data` used here does not include the `Saglia+2016` data.

Reply:
Indeed. I wasn't entirely sure how this works, I thought that the submodule always points to the current master, but that clearly wasn't the case. It should be updated now.